hack/.bin/darwin-amd64/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Out of curiosity, are files in `hack/.bin` necessary for us to deploy ipfix-collector?

Reply:
You're right. I shouldn't commit those to the repo. The verify-helm.sh will install Helm locally.